index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+  <title>Hello Microverse</title>
+</head>
+<body>
+  <h3 class="header">Hello Microverse!</h3>
```

Comment:
checking the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), this heading was supposed to be an h1. Would you kindly change it to H1?,  Having the main heading of your website as an h1, is a best practice as it helps with [SEO](https://www.silverdisc.co.uk/blog/2020/11/06/how-important-are-h1-tags-seo) Also following guidelines strictly will help you in future where clients will have specific features they would want to be implemented in their projects.

Reply:
Thank you, @Kalunge. I will do just that.